### PR TITLE
feat: add initial PPE documentation

### PR DESCRIPTION
**Pull request by mhamas**
This PR is adding initial documentation for pay-per-event Actor pricing model. The product is very young. As it evolves, we will keep updating the documentation.

This is end-user facing documentation. We will write separate documentation for creators once the pricing model becomes available to external developers.

### CODE REVIEW

File: sources/platform/actors/running/store.md [hunk 1]
```diff
@@ -138,6 +141,44 @@ at the bottom of the [Usage tab](https://console.apify.com/billing).
...
+#### What happens when I interact with the dataset after the run finishes?
+You would still pay for all interactions after the Actor run finishes, same as for pay per result Actors.
+
+#### Do I pay for the storage of results on the Apify platform?
+You would still pay for the long term storage of results, same as for pay per result Actors.
+
+#### Where do I see how much I was charged for the pay per result Actors?
+Similarly to pay per result Actors, you can see how much you have been charged on your invoices, and on the [Usage tab](https://console.apify.com/billing) of the Billing section in the Console.
+
+![Pay per event actor - historical usage tab](./images/store/pay_per_event_historical_usage_tab.png)
```
Comment (TC-MO) [inline sources/platform/actors/running/store.md:168]:
```suggestion
![Pay per event Actor - historical usage tab](./images/store/pay_per_event_historical_usage_tab.png)
```

File: sources/platform/actors/running/store.md [hunk 2]
```diff
@@ -138,6 +141,44 @@ at the bottom of the [Usage tab](https://console.apify.com/billing).
...
+
+You can also see the cost of each run on the run detail itself.
+
+![Pay per event actor - run detail](./images/store/pay_per_event_price_on_run_detail.png)
```
Comment (TC-MO) [inline sources/platform/actors/running/store.md:172]:
```suggestion
![Pay per event Actor - run detail](./images/store/pay_per_event_price_on_run_detail.png)
```

File: sources/platform/actors/running/store.md [hunk 3]
```diff
@@ -138,6 +141,44 @@ at the bottom of the [Usage tab](https://console.apify.com/billing).
...
+
+#### Can I put a cap on a cost of a single Actor run?
+Yes, when starting an Actor run, you can define the maximum limit on the cost of that run. When the Actor reaches the defined limit, it should terminate gracefully. Even if it didn't, for any reason, and kept producing results, we make always sure you are never charged more that your defined limit.
+
+![Pay per event actor - max charge per run](./images/store/pay_per_event_price_on_run_detail.png)
```
Comment (TC-MO) [inline sources/platform/actors/running/store.md:177]:
```suggestion
![Pay per event Actor - max charge per run](./images/store/pay_per_event_price_on_run_detail.png)
```

File: sources/platform/actors/running/store.md [hunk 4]
```diff
@@ -20,19 +20,21 @@ Anyone is welcome to [publish Actors](/platform/actors/publishing) in the store,
...
 ## Pricing models
 
-All Actors in [Apify Store](https://apify.com/store) fall into one of the three pricing models:
+[//]: # (TODO: link platform usage docs)
+All Actors in [Apify Store](https://apify.com/store) fall into one of the four pricing models:
 
 1. [**Rental**](#rental-actors) - to continue using the Actor after the trial period, you must rent the Actor from the developer and pay a flat monthly fee in addition to the costs associated with the platform usage that the Actor generates.
 2. [**Pay per result**](#pay-per-result) - you do not pay for platform usage the Actor generates and instead just pay for the results it produces.
-3. [**Pay per usage**](#pay-per-usage) - you can run the Actor and you pay for the platform usage the Actor generates.
+3. [**Pay per event**](#pay-per-event) - you do not pay for platform usage the Actor generates and instead just pay for the specific events the Actor creator defines, such as for generating a single result or starting the Actor.
```
Comment (TC-MO) [inline sources/platform/actors/running/store.md:28]:
```suggestion
3. [**Pay per event**](#pay-per-event) - you can run the Actor and you do not pay for platform usage the Actor generates. Instead you pay for the specific events the Actor creator defines, such as for generating a single result or starting the Actor.
```

**Review by TC-MO — CHANGES_REQUESTED** [review]
few suggestions

**Comment by mhamas** [comment]
@TC-MO I applied suggestions. Can you please take another look? Thx

File: sources/platform/actors/running/store.md [hunk 5]
```diff
@@ -138,6 +141,44 @@ at the bottom of the [Usage tab](https://console.apify.com/billing).
 
 ![Actor pay-per-result cost](./images/store/pay_per_result_actor_items.png)
 
+### Pay per event
+Pay per event Actor pricing model is very similar to the pay per result model. You still do not pay the underlying platform usage. Instead of results, you pay for specific events defined by the creator of the Actor. These events will vary between Actors, and will always be described, together with their pricing, on each Actor. Example events might be producing a single result, doing a unit piece of work (e.g. uploading a file) or starting an Actor.
+
+#### Pay per event Actors - Frequently Asked Questions
```
Comment (TC-MO) [inline sources/platform/actors/running/store.md:147]: add blank line after headings that will fix the markdownlint issues

**Review by TC-MO — APPROVED** [review]
Just one comment to fix the markdownlint issues otherwise looks good

**Pull request merged**